.taskcluster.yml

```diff
@@ -22,7 +22,7 @@ tasks:
 
     scopes: [
       "queue:create-task:lowest:{{ taskcluster.docker.provisionerId }}/deepspeech-worker",
-      "queue:create-task:lowest:{{ taskcluster.docker.provisionerId }}/deepspeech-win",
+      "queue:create-task:lowest:{{ taskcluster.docker.provisionerId }}/deepspeech-win-b",
```

Comment:
it is my understanding that `-b` is intended for "beta". If it's working well and not regressing current `r1.13` we should rather (at the expense of a new deploy, yes) update `deepspeech-win` instance and do not need this commit at all.

Reply:
Right, this was for testing, but at the same time I'm tired of having to wait for redeploys, then the TF PR, then the DS PR...